Controls/Button.js

```diff
@@ -0,0 +1,63 @@
+const Control = require("AXM/Controls/Control");
+const DOM = require("AXM/DOM");
+const { Icon } = require("AXM/Iconography");
+
+class Help extends Control {
+    constructor() {
+        super();
+    }
+
+    createHtml() {
+        return new Icon("fa-question-circle");
+    }
+}
+
+class Button extends Control {
```

Comment:
It was meant mostly as a substitute for `Controls.Button` using classes, so it can be called using JSX as `<Button />` but you're right it's not used within axiom. It can be used within apps using axiom.

Reply:
I'll give it a try as soon as possible in MR.